scripts/tableGenerator.js

```diff
@@ -1578,25 +1578,20 @@ function generateCart() {
         message += `${name} : ${item.quantity}Kg \n`;
     });
     if (currentOrders.length === 0) {
```

Comment:
If the `currentOrders.length === 0`, we dont need to generate the message at all. Try to put this check at the beginning of the function and do a  `return null`.

Keeping this check early can avoid the unnecessary calculations

Reply:
We need to generate the message to tell the customer to choose something. 

I don't think anything is getting messed up with unnecessary calculations. It is working fine, I checked.